core/arch/arm32/plat-vexpress/conf.mk

```diff
@@ -45,6 +45,8 @@ ifeq ($(CFG_CRYPTO_SHA1_ARM32_CE),y)
 CFG_WITH_VFP := y
 endif
 
+CFG_WITH_PRNG_SOFTWARE ?= y
```

Comment:
Sounds better as `CFG_WITH_SOFTWARE_PRNG`. Also it would make more sense to make it the default, because this is the only implementation you can readily have before you implement a driver for a HW generator.